fpdf/fpdf.py

```diff
@@ -2411,6 +2420,16 @@ def rotation(self, angle, x=None, y=None):
             )
             yield
 
+    @check_page
+    @contextmanager
+    def skew_text(self, ax=None, ay=None):
+        epsilon = 1e-5
+        x = math.tan((ax + epsilon) * (math.pi / 180))
+        y = math.tan((ay + epsilon) * (math.pi / 180))
+        with self.local_context():
+            self.skew = (y, x)
```

Comment:
Shouldn't `self.skew` be restored when we exit the context?

Reply:
I'm pretty sure that `self.local_context()` deals with that using `self._pop_local_stack()` and `self._push_local_stack()` no? For example if I do this:  
```
with pdf.skew_text(0, 7):
        with pdf.skew_text(0, 15):
            pdf.cell(txt="text skewed on the y-axis")
            pdf.ln(30)
        pdf.cell(txt="this should also be skewed on the y-axis")
        pdf.ln(30)
pdf.cell(txt="this should not be skewed")
```
I get the following:  
![Screenshot from 2022-12-17 17-02-14](https://user-images.githubusercontent.com/30405025/208248201-ae63dec7-54af-49d2-848d-0e3aa0dcf2f2.png)  
which seems alright

Reply:
Yes, you're right! Perfect